### PR TITLE
Add pimo tutorial advanced i (fixed)

### PR DESCRIPTION
**Pull request by jpcbertoldo**
## 📝 Description

How to select representative samples with AUPIMO scores and how to visualize AUPIMO's bounds on heatmaps.

## ✨ Changes

Select what type of change your PR is:

- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔨 Refactor (non-breaking change which refactors the code base)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] 📚 Documentation update
- [ ] 🔒 Security update

## ✅ Checklist

Before you submit your pull request, please make sure you have completed the following steps:

- [x] 📋 I have summarized my changes in the [CHANGELOG](https://github.com/openvinotoolkit/anomalib/blob/main/CHANGELOG.md) and followed the guidelines for my type of change (skip for minor changes, documentation updates, and test enhancements).
- [ ] 📚 I have made the necessary updates to the documentation (if applicable).
- [x] 🧪 I have written tests that support my changes and prove that my fix is effective or my feature works (if applicable).

For more information about code review checklists, see the [Code Review Checklist](https://github.com/openvinotoolkit/anomalib/blob/main/docs/source/markdown/guides/developer/code_review_checklist.md).

### CODE REVIEW

**Comment by jpcbertoldo** [comment]
So i had to remove the outputs of the cells that print images because the notebook would be too big (8mb), and the 500kb size limit blocked my commit.

It's kinda shame not to have those images there cause it's like half of the notebook's message.

Is there a trick to work around this? 
Or is it ok to do something dirty like save to jpg and reload with an IPython display function, for example ?

**Comment by jpcbertoldo** [comment]
> So i had to remove the outputs of the cells that print images because the notebook would be too big (8mb), and the 500kb size limit blocked my commit.

Update: reducing the images to 96x96 (instead of 256x256) makes the file have 1.8mb. Still above the limit but would that be ok?

**Comment by jpcbertoldo** [comment]
@ashwinvaidya17 can i get a review on this one?

it's teaching and documenting one of those use cases that disappeared in the refactor

**Review by samet-akcay — APPROVED** [review]
lgtm, thanks

**Comment by jpcbertoldo** [comment]
> lgtm, thanks

do you have a suggestion about including the images in the notebook?
would it be ok to commit a big file (<10mb) if it's in `anomalib/notebooks`?

File: CHANGELOG.md [hunk 1]
```diff
@@ -120,6 +120,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+- Add `AUPIMO` tutorials notebooks in https://github.com/openvinotoolkit/anomalib/pull/2330 and https://github.com/openvinotoolkit/anomalib/pull/2336
```
Comment (ashwinvaidya17) [inline CHANGELOG.md:123]: @jpcbertoldo shouldn't this be under unreleased?

Comment (jpcbertoldo) [reply CHANGELOG.md:123]: you are right, my bad i didn't pay attention!
i will correct it in the next one https://github.com/openvinotoolkit/anomalib/pull/2348

Comment (ashwinvaidya17) [reply CHANGELOG.md:123]: No worries! I missed it in the review as well

**Pull request merged**